R/clineshiftScore.R

```diff
@@ -0,0 +1,91 @@
+# clineShiftScore.R
+
+#' \code{clineShiftScore} Generate a cline shift score between two sequences
+#' using a reference distance map and an observed distance map for both
+#' sequences.
+#'
+#' Generate a cline shift score between two sequences. The clineshift score is
+#' a distance score between the expected residue index and the observed residue
+#' index.
+#'
+#' @param refMapA A numeric vector for the distance map of reference sequence A.
+#' @param refMapB A numeric vector for the distance map of reference sequence B.
+#' @param testMapA A numeric vector for the distance map of test sequence A.
+#' @param testMapB A numeric vector for the distance map of test sequence B.
+#' @param epsilon A scoring parameter. Defaults to 0.2.
+#' @return The clineshift score between two sequences.
+#'
+#' @examples
+#' refA <- c("A", "V", "T", "-", "-", "D")
+#' refB <- c("A", "V", "T", "F", "F", "D")
+#' testA <- c("A", "V", "-", "-", "T", "D")
+#' testB <- c("A", "V", "T", "F", "F", "D")
+#' refMap <- generatePairMap(refA, refB)
```

Comment:
generatePairMap(refA, testA?)

Reply:
Pair maps are between two aligned sequences.

Reply:
On the other hand, line 24 was wrong.